recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.1" %}
+{% set sha256 = "300cdbbaa9108879f7ec22d9cf6210ca2700a3ef99c356a5c00fbf3aeb3a4713" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
```

Comment:
For adding the CUDA compilation bits, please see [this comment]( https://github.com/conda-forge/nvcc-feedstock/issues/8#issuecomment-536009949 ).

Reply:
yes, I have the bits locally for testing.